public/test/setupTests.ts

```diff
@@ -1,13 +1,13 @@
 import '@testing-library/jest-dom';
 import i18next from 'i18next';
-import failOnConsole from 'jest-fail-on-console';
+// import failOnConsole from 'jest-fail-on-console';
 import { initReactI18next } from 'react-i18next';
 
 import { matchers } from './matchers';
 
-failOnConsole({
-  shouldFailOnLog: process.env.CI ? true : false,
-});
+// failOnConsole({
+// shouldFailOnLog: true,
+// });
```

Comment:
Do we want to disable this?

Reply:
I think this is no longer needed, it only fails on CI now

Reply:
fixed

Reply:
It actually reverted it https://github.com/grafana/grafana/pull/59901